### ISSUE TITLE
We seem to have an infinite cycle of AFFY processor jobs

### ISSUE THREAD

**Issue by arielsvn** [AlexsLemonade/refinebio#1818]
### Context

We should not be processing anything, but we have several jobs running

### Problem or idea

These are some of the processor jobs running:

```
$ nomad status | grep dispatch | grep running
AFFY_TO_PCL_0_2048/dispatch-1571758727-75294e36  batch                50        running  2019-10-22T15:38:47Z
AFFY_TO_PCL_0_2048/dispatch-1571758732-a694b5bf  batch                50        running  2019-10-22T15:38:52Z
AFFY_TO_PCL_0_2048/dispatch-1571758779-f810c083  batch                50        running  2019-10-22T15:39:39Z
AFFY_TO_PCL_0_2048/dispatch-1571758788-d95c698f  batch                50        running  2019-10-22T15:39:48Z
```

They doesn't seem to have any `start_date` in the DB. That's why they don't appear in the dashboard.

```
data_refinery=> select * from processor_jobs
data_refinery-> where nomad_job_id='AFFY_TO_PCL_0_2048/dispatch-1571758727-75294e36'
data_refinery->     or nomad_job_id='AFFY_TO_PCL_0_2048/dispatch-1571758732-a694b5bf'
data_refinery->     or nomad_job_id='AFFY_TO_PCL_0_2048/dispatch-1571758779-f810c083'
data_refinery->     or nomad_job_id='AFFY_TO_PCL_0_2048/dispatch-1571758788-d95c698f';
    id    | pipeline_applied | no_retry | ram_amount | volume_index | start_time | end_time | success |                  nomad_job_id                   | num_retries | retried | worker_id | worker_version | failure_reason |          created_at           |         last_modified         | retried_job_id 
----------+------------------+----------+------------+--------------+------------+----------+---------+-------------------------------------------------+-------------+---------+-----------+----------------+----------------+-------------------------------+-------------------------------+----------------
 29304326 | AFFY_TO_PCL      | f        |       2048 | 0            |            |          |         | AFFY_TO_PCL_0_2048/dispatch-1571758779-f810c083 |           0 | f       |           |                |                | 2019-10-22 15:39:39.867285+00 | 2019-10-22 15:39:39.896467+00 |               
 29304327 | AFFY_TO_PCL      | f        |       2048 | 0            |            |          |         | AFFY_TO_PCL_0_2048/dispatch-1571758788-d95c698f |           0 | f       |           |                |                | 2019-10-22 15:39:48.707007+00 | 2019-10-22 15:39:48.737365+00 |               
 29304324 | AFFY_TO_PCL      | f        |       2048 | 0            |            |          |         | AFFY_TO_PCL_0_2048/dispatch-1571758727-75294e36 |           0 | f       |           |                |                | 2019-10-22 15:38:47.173538+00 | 2019-10-22 15:38:47.212877+00 |               
 29304325 | AFFY_TO_PCL      | f        |       2048 | 0            |            |          |         | AFFY_TO_PCL_0_2048/dispatch-1571758732-a694b5bf |           0 | f       |           |                |                | 2019-10-22 15:38:52.333198+00 | 2019-10-22 15:38:52.360586+00 |               
(4 rows)
```

After a few minutes, these jobs are also removed from the DB.

```
data_refinery=> select * from processor_jobs
where nomad_job_id='AFFY_TO_PCL_0_2048/dispatch-1571758727-75294e36'
    or nomad_job_id='AFFY_TO_PCL_0_2048/dispatch-1571758732-a694b5bf'
    or nomad_job_id='AFFY_TO_PCL_0_2048/dispatch-1571758779-f810c083'
    or nomad_job_id='AFFY_TO_PCL_0_2048/dispatch-1571758788-d95c698f';
 id | pipeline_applied | no_retry | ram_amount | volume_index | start_time | end_time | success | nomad_job_id | num_retries | retried | worker_id | worker_version | failure_reason | created_at | last_modified | retried_job_id 
----+------------------+----------+------------+--------------+------------+----------+---------+--------------+-------------+---------+-----------+----------------+----------------+------------+---------------+----------------
(0 rows)
```

### Solution or next step

Not sure what is happening looks like the processor job don't see the downloaded files and they trigger new downloader jobs, which in turn don't download the files correctly and trigger new processor jobs continuing the cycle.

We need to investigate more.

**Comment by arielsvn** [comment]
Also found another job without `start_time`

```
    id    | pipeline_applied | no_retry | ram_amount | volume_index | start_time |           end_time            | success |                nomad_job_id                | num_retries | retried | worker_id | worker_version |                                                failure_reason                                                |          created_at           |         last_modified         | retried_job_id 
----------+------------------+----------+------------+--------------+------------+-------------------------------+---------+--------------------------------------------+-------------+---------+-----------+----------------+--------------------------------------------------------------------------------------------------------------+-------------------------------+-------------------------------+----------------
 29309148 | NO_OP            | f        |      32768 | 0            |            | 2019-10-22 18:43:21.762632+00 | t       | NO_OP_0_32768/dispatch-1571762522-0387877d |           1 | f       |           |                | Sample has a good computed file, it must have been processed, so it doesn't need to be downloaded! Aborting! | 2019-10-22 16:42:02.520194+00 | 2019-10-22 18:43:21.762658+00 |               
(1 row)
```

**Comment by arielsvn** [comment]
Found the problem with this. We remove archive files in the GEO downloader after we extract the files we need. However, some original files were incorrectly marked as archives when they were surveyed. I'm going to push a fix for this with #1857 

In total, we had 603 original files incorrectly marked as archives. Fixed them by running the following in prod:

```py
>>> OriginalFile.objects.filter(source_filename__iendswith='.CEL', is_archive=True).count()
603
>>> OriginalFile.objects.filter(source_filename__iendswith='.CEL', is_archive=True).update(is_archive=False)
603
```
